packages/react-ui/src/widgets/WrapperWidgetUI.js

```diff
@@ -30,79 +30,95 @@ Actions props must have this format:
 ];
 */
 
-const useStyles = makeStyles((theme) => ({
-  root: {
+const Root = styled(Box,{
+  shouldForwardProp: (prop) => prop !== 'margin'
+})(({ theme, margin }) => {
+  return {
+    margin: 0,
     position: 'relative',
     maxWidth: '100%',
-    padding: ({ margin }) => (margin !== undefined ? margin : theme.spacing(2, 2.5))
-  },
-  loading: {
-    position: 'absolute',
-    top: 0,
-    left: 0,
-    width: '100%',
-    height: theme.spacing(0.25)
+    padding: margin !== undefined ? margin : theme.spacing(2, 2.5)
+  }
+});
+
+const LoadingBar = styled(LinearProgress)(({ theme }) => ({
+  position: 'absolute',
+  top: 0,
+  left: 0,
+  width: '100%',
+  height: theme.spacing(0.25)
+}));
+
+const Header = styled(Grid, {
+  shouldForwardProp: (prop) => prop !== 'expanded'
+})(({ theme, expanded = true }) => ({
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'flex-start',
+  justifyContent: 'space-between',
+  width: '100%',
+  ...(expanded ? { minHeight: theme.spacing(3) } : { height: theme.spacing(3) }),
+  padding: 0
+}));
+
+const HeaderButton = styled(Button)(({ theme, expandable = true }) => ({
+  flex: 1,
+  padding: 0,
+  alignItems: 'flex-start',
+  justifyContent: 'flex-start',
+  cursor: expandable ? 'pointer' : 'default',
```

Comment:
This 'expandable' prop should be also caught in shouldForwardProps, right?

Reply:
done!